Source/Core/Core/PowerPC/Jit64Common/Jit64AsmCommon.cpp

```diff
@@ -8,11 +8,11 @@
 #include "Common/CPUDetect.h"
 #include "Common/CommonTypes.h"
 #include "Common/EnumUtils.h"
-#include "Common/FloatUtils.h"
 #include "Common/Intrinsics.h"
 #include "Common/JitRegister.h"
 #include "Common/x64ABI.h"
 #include "Common/x64Emitter.h"
+#include "Core/FloatUtils.h"
```

Comment:
The changes you made to the fres routine in this file seem to have gone missing. (Note that Jit64 fres has no unit tests, so this mistake doesn't show up as any test failing.)